content/api/commands/clearalllocalstorage.md

```diff
@@ -0,0 +1,74 @@
+---
+title: clearAllLocalStorage
+---
+
+Clear localStorage data for all origins with which the test has interacted.
```

Comment:
```suggestion
Clear [localStorage](https://developer.mozilla.org/en-US/docs/Web/API/Window/localStorage) data for all origins with which the test has interacted.
```

Reply:
Addressed in https://github.com/cypress-io/cypress-documentation/pull/4876/commits/1b31d3d09ba10bea4afcd9cdc34af0fd7480bd79